action-server/tests/actions/test_test_navigation_form.py

```diff
@@ -165,15 +166,17 @@ def tearDown(self):
         self.geocoding_patcher.stop()
         self.testing_locations_patcher.stop()
 
-    def _set_geocode(self, geocode=GEOCODE, exception=False):
+    def _set_USER_COORDINATES(self, USER_COORDINATES=USER_COORDINATES, exception=False):
```

Comment:
`USER_COORDINATES` en majuscule: j'ai l'impression que tu as fait un search-and-replace qui a mal tourné ici.

Reply:
XD En effet